pkg/distro/fedora/images.go

```diff
@@ -345,6 +345,9 @@ func diskImage(workload workload.Workload,
 
 	img.Filename = t.Filename()
 
+	d := t.arch.distro
+	img.FirstBoot = common.VersionGreaterThanOrEqual(d.osVersion, VERSION_FIRSTBOOT)
```

Comment:
I wish we had an easy way to unit test this as well but we don't currently so its fine.